doc/developing/build.rst

```diff
@@ -0,0 +1,52 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _developing/build:
+
+****************
+Build and deploy
+****************
+
+.. highlight:: bash
+
+TBC
+
+Building and deploying Connect centrally
```

Comment:
@henriknorin-ftrack 

As we have it, might be better to reference the existing connect's documentation.